src/node/build/index.ts

```diff
@@ -391,12 +392,13 @@ export async function build(options: BuildConfig): Promise<BuildResult> {
         modulesOptions: cssModuleOptions
       }),
       // vite:asset
-      createBuildAssetPlugin(
+      createBuildAssetPlugin({
         root,
-        publicBasePath,
+        publicBase: publicBasePath,
         assetsDir,
-        assetsInlineLimit
-      ),
+        inlineLimit: assetsInlineLimit,
+        include: (file) => resolver.isAssetRequest(file)
```

Comment:
This can be Simplified as `include:  resolver.isAssetRequest`

Reply:
I'm worried about the unbound call, in case the implementation of `isAssetRequest` changes. What do you think about changing this param to `resolver` itself?